python/dgl/distributed/rpc.py

```diff
@@ -78,7 +235,6 @@ def register_service(service_id, req_cls, res_cls=None):
     if res_cls is not None:
         RESPONSE_CLASS_TO_SERVICE_ID[res_cls] = service_id
     SERVICE_ID_TO_PROPERTY[service_id] = (req_cls, res_cls)
-    # TODO: C registry
```

Comment:
we'll need to register C callback later, right?

Reply:
Yes.